### ISSUE TITLE
implement the Scripts To Rule Them All interface 

### ISSUE THREAD

**Issue by sam-qordoba** [CodeForPhilly/chime#38]
Implement the [Scripts To Rule Them All](https://github.com/github/scripts-to-rule-them-all) interface for developers:

```
script/server — Brings a fully working environment up at localhost:[port]
script/setup — Run after script/server to initialize local database with seed data
script/update — Run after changing code or switching branches to refresh server
```

An example of this implementation can be found at [CfP's Prevention Point project](https://github.com/CodeForPhilly/prevention-point/tree/master/script)

@garysieling  has a WIP here- will finish testing later: https://github.com/garysieling/chime/tree/feature/36-scripts-to-rule-them-all

**Comment by garysieling** [comment]
My changes  + others are in - verified the scripts work as is

**Comment by themightychris** [comment]
Merged in via #35 and then refined in 2f54f0c572df1b9b78861a9e171e15cb380c2e3c
